ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -1038,22 +1107,39 @@ export const DataAccessRequirements = fp.flow(withProfileErrorModal)(
       onMount();
     }, []);
 
-    // handle the route /nih-callback?token=<token>
-    // handle the route /ras-callback?code=<code>
     const query = useQuery();
+
+    // handle the route /nih-callback?token=<token>
```

Comment:
group these by param

Reply:
I had an idea that it might be fun to turn these into [custom hooks](https://reactjs.org/docs/hooks-custom.html) but I'm splitting that off into #6472